### PR TITLE
Fix rockcraft.yaml

### PR DESCRIPTION
**Pull request by nsklikas**
Fix `rockcraft.yaml`:
- Fix license.
- Replace `entrypoint` with `services`

### CODE REVIEW

**Review by gruyaume — APPROVED** [review]

File: rockcraft.yaml [hunk 1]
```diff
@@ -6,10 +6,15 @@ summary: Canonical Identity platform login UI
...
+license: GPL-3.0
 platforms:
   amd64:
-entrypoint: ["identity_platform_login_ui"]
+
+services:
+  login-ui:
+    override: replace
+    command: identity_platform_login_ui
+    startup: enabled
```
Comment (natalian98) [inline rockcraft.yaml:17]: Why we need to replace entrypoint with services? Will it not interfere with `services` in [pebble layer](https://github.com/canonical/identity-platform-login-ui-operator/blob/main/src/charm.py#L129)?

File: rockcraft.yaml [hunk 2]
```diff
@@ -9,7 +9,12 @@ description: |
 license: GPL-3.0
```
Comment (natalian98) [inline rockcraft.yaml:9]: could there be a problem with your branch? I see the license is changed in both commits, but overall in PR's "Files changed" it's not modified

File: rockcraft.yaml (hunk 1 above)
Comment (massigori) [reply rockcraft.yaml:17]: @natalian98 that seems correct as per the layer specification https://github.com/canonical/pebble#layer-specification

File: rockcraft.yaml (hunk 2 above)
Comment (nsklikas) [reply rockcraft.yaml]: Ok this is very weird, I force pushed my commits again and it seems fixed now.

**Review by natalian98 — DISMISSED** [review]

**Review by bencekov — APPROVED** [review]
Looks good.

**Pull request merged**